Description: fix vs
 .
 handylinux-desktop (2.5.1-3) jessie; urgency=medium
 .
   * ident fix in info4forum
Author: team handylinux <dev@handylinux.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- handylinux-desktop-2.5.1.orig/handylinux_version
+++ handylinux-desktop-2.5.1/handylinux_version
@@ -1 +1 @@
-HandyLinux-2.5.2
+HandyLinux-2.5.1
